LoginScreen.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+import React, {
+  AppRegistry,
+  Component,
+  StyleSheet,
+  TouchableHighlight,
+  Text,
+  TextInput,  
+  View,
+  Navigator
+} from 'react-native';
+
+
+
+
+var LoginScreen = React.createClass({
```

Comment:
I know three ways to create a React component (http://reactjsnews.com/composing-components/):

- using the old style React.createClass
```jsx
var TickTock = React.createClass({
  getInitialState: function() {
    return {seconds: 0};
  },
  componentDidMount: function() {
    this.setInterval(this.tick, 1000); // Call a method on the mixin
  },
  tick: function() {
    this.setState({seconds: this.state.seconds + 1});
  },
  render: function() {
    return (
      <p>
        React has been running for {this.state.seconds} seconds.
      </p>
    );
  }
});
```

- extending from React.Component (ES6)

```jsx
class HelloMessage extends React.Component {
  render() {
    return <div>Hello {this.props.name}</div>;
  }
}
```
- stateless functions

```jsx
function HelloMessage(props) {
  return <div>Hello {props.name}</div>;
}
```

The ES6 and stateless function is the way to go, if them we can have Higher Order Component and do cool stuff (https://medium.com/@dan_abramov/mixins-are-dead-long-live-higher-order-components-94a0d2f9e750).

Do not use **React.createClass** anymore